app/mailers/user_notifications.rb

```diff
@@ -167,7 +167,7 @@ def account_silenced(user, opts = nil)
         reason: user_history.details,
       )
     else
-      silenced_till = user.silenced_till.in_time_zone(user.user_option.timezone)
+      silenced_till = user.silenced_till.in_time_zone(user.user_option.timezone.presence || "UTC")
```

Comment:
`|| "UTC"` isn't strictly necessary here, but it's better to be explicit. This fallback should only be hit in very rare cases.